src/app/modules/account/AccountManager.ts

```diff
@@ -146,15 +173,32 @@ export default class AccountManager {
         this.mnemonic = deserializedData.mnemonic
         this.masterNode = bip32.fromSeed(bip39.mnemonicToSeedSync(this.mnemonic))
 
-        const accounts =  deserializedData.accounts.map(account => {
+        const accounts = deserializedData.accounts.map(account => {
             return {
                 alias: account.alias,
                 identity: RadixSimpleIdentity.fromPrivate(account.privateKey),
             }
         })
         this.setAccounts(accounts)
     }
-    
+
+    public setActiveAccount(account: WalletAccount) {
+        // Unsubscribe old updates
+        if (transferSubscription) {
+            transferSubscription.unsubscribe()
+        }
+
+        store.commit('setActiveAccount', account)
+
+        // Subscribe to updates
+        transferSubscription = account.identity.account.transferSystem.transactionSubject
+            .subscribe(transactionUpdateSubject)
+    }
+
+    public subscribeToTransferEvents(func: any) {
```

Comment:
The type here could be more precise

Reply:
that one should be removed as it's not being used

Reply:
fixed!